tpcx_bb/xbb_tools/cluster_startup.py

```diff
@@ -106,6 +106,12 @@ def maybe_create_worker_directories(dask_worker):
             initial_pool_size=os.environ.get("BLAZING_INITIAL_POOL_SIZE", None)
         )
 
+    
+    #upload sentiment txt files to all nodes
+    client.upload_file('sentiment_files/negativeSentiment.txt')
```

Comment:
Has this been tested multi-node? There used to be problems with `upload_file` in the past.

I believe this sends the file to a temp directory on the workers. Do the queries run from this directory / does it create the full path of `sentiment_files/filename.txt`? 

If not, would `with open("sentiment_files/negativeSentiment.txt") as fh:` fail?

Reply:
multi-node test setup:
Start dask scheduler, and two dask-cuda-workers on separate workers from a copy of the tpcx-bb repo directory _without_ the `sentiment_files` subfolder.

Then, from a copy of the tpcx-bb repo folder _with_ `sentiment_files` in the current working directory, I ran queries 10, 18, and 19, as well as the full benchmark_runner sweep. They all ran successfully.

Reply:
@randerzander , In my previous tests I saw it only being left in like 1/2 nodes across the 17, so unsure how reflective a two node test is here.

Reply:
Decided the cleaner approach is to just stick the sentiment files in a shared filesystem location and load from there.

For future reference, you can do something like:
```
from dask.distributed import get_worker
def copy_to_cwd(fn):
   local_dir = get_worker.local_directory
   #copy fn to os.getcwd()

client.run(copy_to_cwd('negativeSentiment.txt')
```

But that code in a query implementation felt too messy.